apps/api/src/dao/search.service.ts

```diff
@@ -30,7 +30,7 @@ export class SearchService {
 
     // Check Accounts
     if (this.ethService.isValidAddress(query)) {
-      const account = await this.accountService.findAccountByAddress(query)
+      const account = await this.accountService.findAccountByAddress(query.toLowerCase())
```

Comment:
It's better if we have the methods in the services sanitize their input. This centralises the sanitization and means that you don't have to remember to do it with new api calls for example.